docs/src/differentiable_programming.md

```diff
@@ -360,7 +360,54 @@ sol = solve(ode, BS3(), save_everystep=false)
 Trixi.integrate(energy_total, sol.u[end], semi)
 ```
 do not need any modifications since they are sufficiently generic (and enough effort
-has been spend to allow general types inside thee calls).
+has been spend to allow general types inside these calls).
+
+
+### Propagating errors using Measurements.jl
+
+![tutorial_measurements1](https://imgs.xkcd.com/comics/error_bars.png)
```

Comment:
I like the picture! However, we should at least link back to Xkcd or add a (c) Randall Munroe with a link (see also https://xkcd.com/license.html).

Reply:
Good point. I added a link in 420bca0a